### PR TITLE
ENH use azure to get around travis

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   merge_build_host: true  # [win]
-  number: 1
+  number: 2
```
Comment (bgruening) [inline recipe/meta.yaml:17]: just as a question, do you need to bump? only the ppc should be build if you leave it as 1, left?

Comment (beckermr) [reply recipe/meta.yaml:17]: True. We could not bump. There is not a lot of harm here to bump, but I am happy to remove the bump too. I do it by habit now.

Comment (bgruening) [reply recipe/meta.yaml:17]: all good, was just wondering

**Pull request merged**